store/TransactionSlice/transactions.tsx

```diff
@@ -0,0 +1,203 @@
+        success: (props: any) => UpdateOptions;
+        error: ({ props, error }: { props?: any; error: any }) => UpdateOptions;
+    }
+> = {
+    DEFAULT: {
+        pending: () => [
+            <>
+                <Notification title="Transaction Pending" />
+            </>,
+        ],
+        success: () => ({
+            render: <Notification title="Order Submitted" />,
+            type: 'success',
+            isLoading: false,
+        }),
+        error: ({ error }) =>
+            knownTransactionErrors(error) ?? {
+                render: <Notification title="Transaction Failed" />,
+                type: 'error',
+            },
+    },
+    APPROVE: {
+        pending: (props) => [
+            <>
+                <Notification title={`Approving ${props.settlementToken}`} />
+            </>,
+        ],
+        success: (props) => ({
+            render: <Notification title={`${props.settlementToken} Unlocked`} />,
+            type: 'success',
+            isLoading: false,
+            autoClose: AUTO_DISMISS,
+        }),
+        error: ({ props, error }) =>
+            knownTransactionErrors(error) ?? {
+                render: <Notification title={`Unlock ${props.settlementToken} Failed`} />,
+                type: 'error',
+                isLoading: false,
+                autoClose: AUTO_DISMISS,
+            },
+    },
+    COMMIT: {
+        pending: (props) => [
+            <>
+                <CommitPendingNotification {...props} />
+            </>,
+        ],
+        success: (props) => ({
+            render: <CommitSuccessNotification {...props} />,
+            type: 'success',
+            isLoading: false,
+            autoClose: props.nextRebalance * 1000 - Date.now(), // seconds till next rebalance
+        }),
+        error: ({ error }) =>
+            knownTransactionErrors(error) ?? {
+                render: <Notification title={`Transaction Failed`} />,
+                type: 'error',
+                isLoading: false,
+                autoClose: AUTO_DISMISS,
+            },
+    },
+    CLAIM: {
+        pending: (props) => [
+            <>
+                <Notification title={`Claiming ${props.poolName} Tokens`} />,
+            </>,
+        ],
+        success: (props) => ({
+            render: <Notification title={`${props.poolName} Claim Queued`} />,
+            type: 'success',
+            isLoading: false,
+            autoClose: AUTO_DISMISS,
+        }),
+        error: ({ props, error }) =>
+            knownTransactionErrors(error) ?? {
+                render: <Notification title={`Claim from ${props.poolName} Failed`} />,
+                type: 'error',
+                isLoading: false,
+                autoClose: AUTO_DISMISS,
+            },
+    },
+    ARB_ETH_DEPOSIT: {
+        pending: (props) => [
+            <>
+                <Notification title={`Submitting ${props.tokenSymbol} ${props.type} from ${props.networkName}`} />,
+            </>,
```

Comment:
Not a big deal but the fragment is adding a comma to the ui when its displayed. It can be fixed by adding a key to the notification component

Reply:
ahhh cheers, I think it actually that extra comma in the array